packages/react-atlas-core/src/table/Table.js

```diff
@@ -1,61 +1,85 @@
-import React, { PropTypes } from "react";
+import React, {Component, PropTypes} from "react";
 import cx from 'classNames';
+import {BootstrapTable} from "react-bootstrap-table";
 
-const Table = ({ className, children, ...props }) => {
-  const classes = cx(
-    {
-      "table": true
-    }
-  );
-
-  return (
-    <table {...props} styleName={classes} className={cx(className)}>
-      {children}
-    </table>
-  );
-};
+class Table extends Component {
+  render() {
+    const {className, children, styles, ...props} = this.props;
+    
+    return (
+      <BootstrapTable {...props} className={cx(className)}
+                      containerClass={styles['react-bs-table-container']}
+                      tableContainerClass={styles['react-bs-table']}
+                      headerContainerClass={styles['react-bs-container-header']}
+                      bodyContainerClass={styles['react-bs-container-body']}>
+        {children}
+      </BootstrapTable>
+    );
+  }
+}
 
 Table.propTypes = {
```

Comment:
Can you add comments to the different props that can be passed into the table component and explain what they do. These comments will appear on the docsite and users will need the comments to know how to use the Table component.